domains/authorization/Authorization.API/API/Authorization/Controllers/ConsentController.cs

```diff
@@ -53,9 +53,39 @@ public async Task<ActionResult> GetConsent()
         var queryResult = await mediator.Send(new GetUserOrganizationConsentsQuery(identity.Subject.ToString(), identity.OrganizationCvr!));
 
         var response = new UserOrganizationConsentsResponse(
-            queryResult.Result.Select(item => new UserOrganizationConsentsResponseItem(item.ClientName, item.ConsentDate))
+            queryResult.Result.Select(item => new UserOrganizationConsentsResponseItem(item.ClientId, item.OrganizationId, item.ClientName, item.ConsentDate))
         );
 
         return Ok(response);
     }
+
+    /// <summary>
+    /// Deletes a consent, from the organization, which the user is affiliated with.
+    /// </summary>
+    /// <param name="clientId">The ID of the client.</param>
+    /// <param name="organizationId">The ID of the organization.</param>
+    /// <returns>No content if the deletion was successful, Not Found if the consent was not found.</returns>
+    [HttpDelete("api/authorization/consents/{clientId}/{organizationId}")]
```

Comment:
Isnt the organizationId in the jwt token?

Reply:
Another note: 
We always talk about being more use case based rather than rest based. Couldnt this be a prefect example of making our urls more use case based?

Reply:
Can you give an example?

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1236/commits/94b1f55c3688b382e42989a6820e5bde220f891d